README.md

```diff
@@ -154,6 +154,28 @@ spec:
     match:
     - nodeLabel: "node-role.kubernetes.io/worker"
     
+```
```

Comment:
You need rebase on top of  the master. Looks like this have pulled all changes from current master and added to your  PR

Reply:
https://github.com/bnshr/ptp-operator/tree/framework_refinement

Reply:
You branch shows four commit behind.

Reply:
Resolved now!

Reply:
It still showing  merged PR code here ?You should rebase your code on top of the latest

Reply:
Resolved!